src/sass/_globals.scss

```diff
@@ -0,0 +1,26 @@
+ * {
+ margin: 0px;
+ padding: 0px; }
+  body {
+    $fontPrimary;
+    $textPrimary;
+  }
+  .container {
+    max-width: 480px;
+    padding: 0 15px;
+    margin: 0 auto;
+  
+    @media screen and (min-width: 768px) {
```

Comment:
czy na pewno to zadziała, bo masz `@media` w środku `.container` ?

Czy to nie jest tak, że albo są na końcu jako osobny byt, albo jak już w środek `.container` dajesz to dajesz `@include` ?

Mogę się mylić, bo po prostu nie widziałam takiego sposobu, ale jeśli działał u Ciebie w hw-8 to gitówa